src/app/components/BibPage/BibDetails.jsx

```diff
@@ -137,7 +137,7 @@ class BibDetails extends React.Component {
 
       return markup.length === 1
         ? markup.pop()
-        : (<ul>{markup.map(m => (<li key={m}>{m}</li>))}</ul>);
+        : (<ul>{markup.map(m => (<li key={m[0].key}>{m}</li>))}</ul>);
```

Comment:
`m` is an array of React elements, which will always have a key property. I guess I could check for `m[0]` and `m[0].key`, but I'm not sure it will ever be necessary based on how the function work.